tutorial/docker_config/slurm/docker-compose.yml

```diff
@@ -41,7 +41,7 @@ services:
       - var_log_slurm:/var/log/slurm
     ports:
       - "8888"
-      - "8787"
+      - "8787:8787"
```

Comment:
I wouldn't do this to be honest, I thought you said that the URL that we print on `start_slurm` actually works. If you place that into the Dask plugin, it should still work. If we explicitly set 8787, then we are making the assumption that it is available which we have seen not to be necessarily true.

Reply:
Hmm, no that doesn't seem to be true, I was just trying it on my own system, I guess there are some internals that are not expecting the port swap

Reply:
>  If you place that into the Dask plugin, it should still work.

No, this is not the case. `localhost:32XXX` does not work because the lab extension checks that the URL exists **from Python**. That runs on the container; there is no scheduler at `localhost:32XXX` there.

Reply:
The only way I can think of to avoid the overlap with existing stuff on 8787 would be to use some other arbitrarily chosen port, and explicitly set in when making the cluster. Probably easier to ask them not to be running other notebooks at the same time!

Reply:
Ok, yeah, I get it now. Well at least 8787 is a little less likely to be used than 8888